src/app/tests/suites/certification/Test_TC_MOD_1_1.yaml

```diff
@@ -37,29 +37,38 @@ tests:
               type: uint16
 
     - label: "TH reads the FeatureMap attribute from the DUT"
-      disabled: true
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
           value: 1
           constraints:
               type: map8
 
+    #if condition needed #Issue 10988
     - label: "TH reads the AttributeList attribute from the DUT"
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AttributeList"
       response:
+          value: [0, 1, 2, 3, 65528, 65529, 65531, 65532, 65533]
```

Comment:
This is a subset minimum. It could be more attributes, especially if there are vendor-specific ones added. The list cannot be an equality list

Reply:
@tcarmelveilleux we need conditional check support for this step because we don't know weather the particular PICS supported or not for time being i just added mandatory values of AttributeList as per testplan.
![image](https://user-images.githubusercontent.com/102359958/166618712-5ad41215-0b9e-405f-91d4-72b31dd1422e.png)